src/main/resources/db/changelog/changes/04-create-user-roles-table.yaml

```diff
@@ -0,0 +1,26 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-users-roles-table
+      author: voitenko
+      changes:
+        - createTable:
+            tableName: users_roles
+            columns:
+              - column:
+                  name: role_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+                    foreignKeyName: role_foreignKey
```

Comment:
Try to name your constraints globally unique. Here are the conventions: https://repository.prace-ri.eu/git/help/development/database/constraint_naming_convention.md